pyface/tests/test_ui_traits.py

```diff
@@ -348,6 +372,22 @@ def test_defaults(self):
         self.assertEqual(border.left, 0)
         self.assertEqual(border.right, 0)
 
+    def test_unspecified_default(self):
```

Comment:
Needs a different name from the earlier `test_unspecified_default`?

Reply:
different `TestCase` class

Reply:
Whoops, yes. Sorry.